webpack/assets/javascripts/react_app/components/ConfirmModal/index.js

```diff
@@ -53,6 +53,7 @@ const ConfirmModal = () => {
 
   return (
     <Modal
+      ouiaId="app-confirm-modal"
```

Comment:
I wonder if we should ask PF to use the `id` for `ouiaId` by default if it wasn't provided.